src/Products/Annotation/Util/AnnotationMapper.cs

```diff
@@ -48,34 +53,67 @@ public AnnotationDataEntity[] mapForPage(AnnotationInfo[] annotations, int pageN
         /// </summary>
         /// <param name="annotationInfo">AnnotationInfo</param>
         /// <returns>AnnotationDataEntity</returns>
-        public AnnotationDataEntity mapAnnotationDataEntity(AnnotationInfo annotationInfo)
+        public AnnotationDataEntity MapAnnotationDataEntity(AnnotationBase annotationInfo, PageInfo pageInfo)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'MapAnnotationDataEntity' a static method.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)